src/CMakeLists.txt

```diff
@@ -55,11 +55,7 @@ if (ipo_supported)
     set_target_properties(${PROJECT_NAME} PROPERTIES INTERPROCEDURAL_OPTIMIZATION TRUE)
 endif ()
 # set compiler flags depending on compiler
-if (MSVC)
-    target_compile_options(${PROJECT_NAME} PRIVATE "/W4" "/utf-8")
-else ()
-    target_compile_options(${PROJECT_NAME} PRIVATE -Wall -Wextra -Wpedantic)
-
+if (NOT MSVC)
     if (BINDINGS)
```

Comment:
Ah, I missed that until now. You can delete almost everything in here, i.e., everything from 
```
# add coverage compiler and linker flag if COVERAGE is set
```
until (not including)
```
add_library(MQT::${PROJECT_NAME} ALIAS ${PROJECT_NAME})
```
can be removed from here as it is covered by the `project_options` target.